src/account/monster.js

```diff
@@ -40,11 +40,15 @@ exports.monster = {
       address: {
         sqlColumn: 'address',
       },
+      name: {
+        sqlColumn: 'username',
+      },
       unconfirmedBalance: {
         sqlColumn: 'u_balance',
       },
       balance: {
         sqlColumn: 'balance',
+        resolve: row => fromRawLsk(row.balance),
```

Comment:
Maybe better to use the graphql resolvers ?

Reply:
Is it possible to overwrite it again in graphql resolvers?

Reply:
I guess no?